tests/SCF/test_SCF.py

```diff
@@ -17,3 +24,45 @@ def test_calc_initial_density(mol_h2o):
     Duv = SCF.calc_initial_density(mol_h2o)
     assert Duv.sum() == 0.0
     assert Duv.shape == (mol_h2o.nao,mol_h2o.nao)
+
+def test_calc_hcore_matrix():  # checks Hcore matrix
+    Tuv = pickle.load(open("tuv.pkl", "rb"))
+    Vuv = pickle.load(open("vuv.pkl", "rb"))
+    Huv = SCF.calc_hcore_matrix(Tuv, Vuv)
+    assert Huv[0, 0] == -32.57739541261037
+    assert Huv[3, 4] == 0.0
+
+
+def test_calc_fock_matrix(mol_h2o):  # checks fock matrix
+    Huv = SCF.calc_hcore_matrix(Tuv, Vuv)
+    Duv = SCF.calc_initial_density(mol_h2o)
+    Fuv= SCF.calc_fock_matrix(mol_h2o, Huv, eri, Duv)
+    assert Fuv[0,0] == -32.57739541261037
+    assert Fuv[2,5] == pytest.approx(-1.6751501447185015)  # needed because the answers are not exactly equal
+    assert Fuv[5,2] == pytest.approx(-1.6751501447185015)
+
+def test_solve_Roothan_equations(mol_h2o):  # checks Roothan euations (eigen vectors and eigen values)

```

Comment:
Tuv, Vuv should be accessed through the Fixture, not the global variables above.

Reply:
Suv should be accessed through the fixture and not a global variables.